src/view/LoggedInView.java

```diff
@@ -196,8 +197,8 @@ public void propertyChange(PropertyChangeEvent evt) {
         LoggedInState loggedInState = (LoggedInState) evt.getNewValue();
         username.setText(loggedInState.getUsername());
         bio.setText(loggedInState.getBio());
-        spotifyHandle.setText(loggedInState.getSpotifyHandle);
-        topThreeArtists.setText(loggedInState.getTopThreeArtists);
+        spotifyHandle.setText(loggedInState.getSpotifyHandle());
+        topThreeArtists.setText(loggedInState.getTopThreeArtists()); // need to convert this into a string
```

Comment:
Should this be done in the logged in view then?

Reply:
Yes!